diff -upr a/tools/depends/target/ffmpeg/CMakeLists.txt b/tools/depends/target/ffmpeg/CMakeLists.txt
--- a/tools/depends/target/ffmpeg/CMakeLists.txt	2021-05-08 18:20:52.000000000 +0200
+++ b/tools/depends/target/ffmpeg/CMakeLists.txt	2021-08-02 20:35:45.305965332 +0200
@@ -93,6 +93,8 @@ if(EXTRA_FLAGS)
   list(APPEND ffmpeg_conf ${EXTRA_FLAGS})
 endif()
 
+list(APPEND ffmpeg_conf --enable-libdrm --enable-v4l2-request --enable-libudev)
+
 message(STATUS "FFMPEG_CONF: ${ffmpeg_conf}")
 
 set(MAKE_COMMAND $(MAKE))
